src/components/retrieve_from_weaviate/src/main.py

```diff
@@ -0,0 +1,55 @@
+import dask
+import pandas as pd
+from fondant.component import PandasTransformComponent
+from llama_index import ServiceContext, VectorStoreIndex
+from llama_index.embeddings import HuggingFaceEmbedding
+from llama_index.vector_stores import WeaviateVectorStore
+
+import weaviate
+
+dask.config.set({"dataframe.convert-string": False})
```

Comment:
Fondant already sets this, so we shouldn't need this here.

Reply:
done